test/functional/plugin/shada_spec.lua

```diff
@@ -715,6 +722,7 @@ describe('In autoload/shada.vim', function()
         '  # Unexpected enum value: expected one of 0 (CHARACTERWISE), '
         .. '1 (LINEWISE), 2 (BLOCKWISE)',
         '  + rt   type         10',
+        '  + ru   is_unnamed   FALSE',
```

Comment:
Specifically I remember that I check each and every possible key with unexpected type. Also a test where header contains every single key which is defined for any kind of entry. And a test where header contains every single key which is defined for any kind of entry, but with value of another type.